tests/models/test_models.py

```diff
@@ -922,6 +922,50 @@ def test_data_splitter():
         ds.train_dataloader()
         ds.val_dataloader()
 
+    # test that data_loader_kwargs get default drop_last=3 when respective keyword arg is not provided to DataSplitter
+    ds = DataSplitter(adata_manager, train_size=400 / a.n_obs, batch_size=397)
```

Comment:
can all of these tests be moved into the data testing code? and each test should be its own function

Reply:
yes, now implemented but pls see comment below